front/src/store/tv/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,65 @@
+import { tvReducer } from '../reducers';
+import {
+  REQUEST_BASIC_SHOW_INFO_SUCCEEDED,
+  SAVE_EPISODE_DATA,
+  SAVE_SEARCH_QUERY,
+  SET_CALENDAR_EPISODES,
+} from '../actions';
+
+describe('tv reducer', () => {
+  const commonState = {
+    savedQueries: [{ query: '1' }],
+    episodeData: {},
+    basicShowInfo: {},
+    calendarEpisodesForDisplay: [],
+  };
+
+  it('saves query', () => {
+    const action = { type: SAVE_SEARCH_QUERY, payload: { query: '1' } };
+    expect(tvReducer(undefined, action)).toEqual(commonState);
+  });
+
+  it('does not add same query', () => {
+    const initialState = {
```

Comment:
this `initialState` looks the same as commonState above. do we need it?

Reply:
Also I'm not sure if we need to test that it doesn't duplicate the query, so maybe we can remove this test. But let me know if you think we should keep it

Reply:
It is the core logic of the reducer on this action. So I believe we should but I can remove it as well.  initialState is required here because we need to have savedQuery array to contain something to be able to test filtering.

Reply:
Ok sounds good keeping it. Except `initialState` is the same as `commonState` above. Can we use commonState?